docs/source/using/addons.adoc

```diff
@@ -26,13 +26,15 @@ The following example shows the contents of an add-on file, including the name a
 # Name: anyuid                                                                          // <1>
 # Description: Allows authenticated users to run images under a non pre-allocated UID   // <2>
 # Required-vars: ACME_TOKEN                                                             // <3>
+# OpenShift-Version: v3.6.0+                                                            // <4>
```

Comment:
What is the syntax here? Exact version match or supporting something like >v3.6.0 and <v3.6.0 as well? Is the '+' in v3.6.0+ intentional and what does it mean? Is + the same as >?

Reply:
I have same questions as @hferentschik.

Reply:
I see two options (and it depends really on how we think this is going to be used). 

*  Using this meta tag means always >= the specified version. In this case, we should probably name the tag 'Min-OpenShift-Versoin'. 
* Have some form of version range semantic aka allow the use of <> or maybe even things like ~. But this needs to be documented